src/main/java/LacksDataBaseProject/AddAndRemoveLack.java

```diff
@@ -0,0 +1,37 @@
+package LacksDataBaseProject;
+
+import java.util.Scanner;
+
+public class AddAndRemoveLack {
+    static Scanner input = new Scanner(System.in);
+
+    public static void addlack(User user){
```

Comment:
Tutaj (chyba... bo nie wiem czy tak jest dobrze ale tak zrozumiałem Piotrka:)) powinniśmy tego Usera sprawdzać czy zgodnie z rolą on może tworzyć ten Lack, a jak nie to wypad, bo w tym momencie on nie jest do niczego wykorzystywany.
No i albo lack przekazujemy w argumentach metody albo tak jak ty zrobiles tu i przy tworzeniu Usera - ze zbieramy te pola niezbędne po kolei i na koncu robimy new Lack, ale wtedy musielibysmy mocno ta moja metode createMissingLackData zmieniac i jej "podmetody" zeby to jakos spojne bylo.
No i oczywiscie trzeba bedzie jakos polaczyc twoja AddLack z moja createMissingLackData  w jednym ciagu bo dopiero wtedy uzyskamy taki efekt, że w jednym miejscu powstaje konkretny Lack.